cinema_service/settings.py

```diff
@@ -86,12 +87,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
+        "USER": os.environ.get("POSTGRES_USER"),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD"),
+        "HOST": os.environ.get("POSTGRES_HOST"),
+        "PORT": os.environ.get("POSTGRES_PASSWORD"),
```

Comment:
The `PORT` setting in the `DATABASES` configuration is incorrectly using `os.environ.get("POSTGRES_PASSWORD")`. It should be `os.environ.get("POSTGRES_PORT")` to correctly retrieve the port number from the environment variables.